src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -880,7 +868,10 @@ public static void SaveBackupStatus(PeriodicBackupStatus status, DocumentDatabas
                     logger.Operations(message, e);
 
                 backupResult?.AddError($"{message}{Environment.NewLine}{e}");
+                documentDatabase.ConfigurationStorage.BackupHistoryStorage.StoreBackupHistoryEntries(command);
             }
+
+            documentDatabase.ConfigurationStorage.BackupHistoryStorage.StoreBackupDetails(backupResult, status);
```

Comment:
would it make sense to unify this with `RetractTemporaryStoredBackupHistoryEntries` on top in a single write-tx ?